### PR TITLE
Alt text for images

### PR DESCRIPTION
**Pull request by jsmcgd**

### CODE REVIEW

File: web/app/themes/ppj/page-footer.php [hunk 1]
```diff
@@ -43,7 +43,9 @@
                 <div class="footer">
                     <div class="footer__container">
                         <?= $footerContent; ?>
-                        <div class="footer__logo"></div>
+                        <div class="footer__logo">
+                            <img src="/app/themes/ppj/dest/img/svg/logo.svg" alt="HM Prison and Probation Services logo"/>
```
Comment (ollietreend) [inline web/app/themes/ppj/page-footer.php]: It'd be preferable to use WordPress' [`get_template_directory_uri`](https://developer.wordpress.org/reference/functions/get_template_directory_uri/) function here, rather than hardcoding the URL path to the theme.

```html
<img src="<?= get_template_directory_uri() ?>/dest/img/svg/logo.svg">
```

Comment (ollietreend) [inline web/app/themes/ppj/page-footer.php]: The text used here is "Services" – but elsewhere you've used "Service".

Is HMPPS "Services" or "Service"? This should be consistent throughout all alt text.

File: web/app/themes/ppj/partials/headerNavigationMenu.php [hunk 2]
```diff
@@ -4,6 +4,15 @@
 
 $modifiedSiteWideNavItems = ppj\markCurrentlySelectedAncestorMenuItem(ppj\navMenuItems('site-wide'));
 $logoTargetUrl = LegNav\legHomeUrl();
+$logoSrc = "/app/themes/ppj/dest/img/svg/logo.svg";
+$logoAltText = "HM Prison and Probation Service logo";
+
+// The logo for the landing page and the Prison Officer leg is the same,
+// however a specific logo is required for the Youth Custody leg
+if (ppj\LegNav\legName() == 'youth-custody') {
+    $logoSrc = "/app/themes/ppj/dest/img/svg/ycs_logo.svg";
```
Comment (ollietreend) [inline web/app/themes/ppj/partials/headerNavigationMenu.php]: [`get_template_directory_uri`](https://developer.wordpress.org/reference/functions/get_template_directory_uri/)

File: web/app/themes/ppj/partials/mobileNavigation.php [hunk 3]
```diff
@@ -32,7 +32,13 @@
 <div class="mobile-nav">
     <div class="mobile-nav__overlay"></div>
     <button class="mobile-nav__close-button"
+            aria-label="Close mobile mobile navigation menu button"
```
Comment (ollietreend) [inline web/app/themes/ppj/partials/mobileNavigation.php]: I think this could be a bit wordy. Would `aria-label="Close Menu"` be better?

Either way, there's a typo – you've repeated "mobile" twice. I also think including the word "mobile" here adds no value, since to the user that's the only navigation menu available to them. Whilst it makes sense to us as developers, I don't think the "mobile" nav needs to be labelled as such to end users.

File: web/app/themes/ppj/partials/mobileNavigation.php [hunk 4]
```diff
@@ -32,7 +32,13 @@
...
             onclick="ppj.closeNavMenu()">
+        <span class="mobile-nav__close-button-text">Close mobile menu</span>
+        <img class="mobile-nav__close-button-image"
+             src="/app/themes/ppj/dest/img/svg/close.svg"
+             alt="Close mobile mobile navigation menu button image"
```
Comment (ollietreend) [inline web/app/themes/ppj/partials/mobileNavigation.php]: Same here – 'mobile' is repeated twice. And I'm not convinced 'mobile' needs to be used at all anyway.

File: web/app/themes/ppj/partials/mobileNavigation.php [hunk 5]
```diff
@@ -32,7 +32,13 @@
 <div class="mobile-nav">
     <div class="mobile-nav__overlay"></div>
     <button class="mobile-nav__close-button"
+            aria-label="Close mobile mobile navigation menu button"
             onclick="ppj.closeNavMenu()">
+        <span class="mobile-nav__close-button-text">Close mobile menu</span>
```
Comment (ollietreend) [inline web/app/themes/ppj/partials/mobileNavigation.php]: This could simply be 'Close menu' – no need for 'mobile'

File: web/app/themes/ppj/partials/videoPlayer.php [hunk 6]
```diff
@@ -25,13 +25,17 @@ class="video-player__cover"
...
+                    aria-labelledby="playButtonIconTitle playButtonIconDesc"
+                    class="video-player__play-button"
                      x="0px"
                      y="0px"
                      viewBox="0 0 138.3 138.3"
                      style="enable-background:new 0 0 138.3 138.3;"
                      role="img"
                 >
+                    <title id="playButtonIconTitle">PlayButton icon</title>
+                    <desc id="playButtonIconDesc">An white triangle pointing to the right, within a circle</desc>
```
Comment (ollietreend) [inline web/app/themes/ppj/partials/videoPlayer.php]: - Missing a space between "Play" and "Button" in the `<title>` tag.
- Should be "a" white triangle, not "an" in the `<desc>` tag.

File: web/app/themes/ppj/src/img/svg/close.svg [hunk 7]
```diff
@@ -1,6 +1,19 @@
 <?xml version="1.0" encoding="iso-8859-1"?>
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Capa_1" x="0px" y="0px" viewBox="0 0 47.971 47.971" style="enable-background:new 0 0 47.971 47.971;" xml:space="preserve" width="512px" height="512px">
-<g>
-	<path d="M28.228,23.986L47.092,5.122c1.172-1.171,1.172-3.071,0-4.242c-1.172-1.172-3.07-1.172-4.242,0L23.986,19.744L5.121,0.88   c-1.172-1.172-3.07-1.172-4.242,0c-1.172,1.171-1.172,3.071,0,4.242l18.865,18.864L0.879,42.85c-1.172,1.171-1.172,3.071,0,4.242   C1.465,47.677,2.233,47.97,3,47.97s1.535-0.293,2.121-0.879l18.865-18.864L42.85,47.091c0.586,0.586,1.354,0.879,2.121,0.879   s1.535-0.293,2.121-0.879c1.172-1.171,1.172-3.071,0-4.242L28.228,23.986z" fill="#FFFFFF"/>
-</g>
+<svg
+  aria-labelledby="closeButtonTitle closeButtonDesc"
```
Comment (ollietreend) [inline web/app/themes/ppj/src/img/svg/close.svg]: I'm not sure if the `aria-labelledby` is useful in self-contained SVG files. But it's probably not causing any harm, either..

File: web/app/themes/ppj/src/img/svg/facebook.svg [hunk 8]
```diff
@@ -1,9 +1,18 @@
...
+  style="enable-background:new 0 0 246.9 246.9;"
+  viewBox="0 0 246.9 246.9"
+  x="0px"
+  xml:space="preserve"
+  xmlns="http://www.w3.org/2000/svg"
+  xmlns:xlink="http://www.w3.org/1999/xlink"
+  y="0px"
+>
+  <title id="facebookIconTitle">Facebook icon</title>
+  <desc id="facebookIconDesc">A navy blue F character on a white background</desc>
```
Comment (ollietreend) [inline web/app/themes/ppj/src/img/svg/facebook.svg]: A better description might be "An icon representing the Facebook logo" – rather than describing the logo itself 🤷🏻‍♂️

File: web/app/themes/ppj/src/vue/FindAJob.vue [hunk 9]
```diff
@@ -143,10 +146,15 @@
...
-                 xml:space="preserve"
+            <svg
+                aria-labelledby="previousPageButtonTitle previousPageButtonDesc"
+                viewBox="0 0 32.9 32.9"
+                x="0px"
+                xml:space="preserve"
+                y="0px"
             >
+              <title id="previousPageButtonTitle">Previous page button</title>
+              <desc id="previousPageButtonDesc">A shape like a less than sign</desc>
```
Comment (ollietreend) [inline web/app/themes/ppj/src/vue/FindAJob.vue]: "An arrow pointing left" might be a more user friendly description of this image.

File: web/app/themes/ppj/src/vue/FindAJob.vue [hunk 10]
```diff
@@ -163,13 +171,16 @@
...
+              aria-labelledby="nextPageButtonTitle nextPageButtonDesc"
+              viewBox="0 0 32.9 32.9"
+              x="0px"
+              y="0px"
             >
-                  <path class="st0" d="M11.7,21.2v-0.9c0-0.2,0.1-0.3,0.2-0.3l7.5-3.4c0.1,0,0.1-0.1,0-0.1L11.9,13c-0.1-0.1-0.2-0.2-0.2-0.3v-0.9
-                    c0-0.2,0.1-0.3,0.3-0.2l8.9,4.1c0.1,0.1,0.2,0.2,0.2,0.4v0.9c0,0.2-0.1,0.3-0.2,0.4l-8.9,4.1C11.9,21.5,11.7,21.4,11.7,21.2z"/>
-
+              <title id="nextPageButtonTitle">Next page button</title>
+              <desc id="nextPageButtonDesc">A shape like a greater than sign</desc>
```
Comment (ollietreend) [inline web/app/themes/ppj/src/vue/FindAJob.vue]: "An arrow pointing right" might be a more user friendly description of this image.

File: web/app/themes/ppj/page-footer.php (hunk 1 above)
Comment (jsmcgd) [reply web/app/themes/ppj/page-footer.php]: done

Comment (jsmcgd) [reply web/app/themes/ppj/page-footer.php]: done

File: web/app/themes/ppj/partials/headerNavigationMenu.php (hunk 2 above)
Comment (jsmcgd) [reply web/app/themes/ppj/partials/headerNavigationMenu.php]: done

File: web/app/themes/ppj/partials/mobileNavigation.php (hunk 3 above)
Comment (jsmcgd) [reply web/app/themes/ppj/partials/mobileNavigation.php]: Duplicates removed. Will discuss text with Josh.

File: web/app/themes/ppj/partials/mobileNavigation.php (hunk 4 above)
Comment (jsmcgd) [reply web/app/themes/ppj/partials/mobileNavigation.php]: Same as above.

File: web/app/themes/ppj/partials/videoPlayer.php (hunk 6 above)
Comment (jsmcgd) [reply web/app/themes/ppj/partials/videoPlayer.php]: done

File: web/app/themes/ppj/src/img/svg/facebook.svg (hunk 8 above)
Comment (jsmcgd) [reply web/app/themes/ppj/src/img/svg/facebook.svg]: Will discuss with Josh.

File: web/app/themes/ppj/src/vue/FindAJob.vue (hunk 9 above)
Comment (jsmcgd) [reply web/app/themes/ppj/src/vue/FindAJob.vue]: Will discuss with Josh.

File: web/app/themes/ppj/src/vue/FindAJob.vue (hunk 10 above)
Comment (jsmcgd) [reply web/app/themes/ppj/src/vue/FindAJob.vue]: Will discuss with Josh.
